main.ts

```diff
@@ -90,13 +107,39 @@ async function ensureNscloudToken() {
   await exec.exec("nsc auth exchange-github-token --ensure=5m");
 }
 
-async function remoteNscBuilderExists(): Promise<boolean> {
+async function nscBuilderStatus(): Promise<boolean> {
+  const { stdout, stderr } = await exec.getExecOutput(
+    `nsc docker buildx status --output=json`,
```

Comment:
Did we release a new `nsc` to all customers?

Reply:
now yes :)